modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -575,6 +575,11 @@ export interface CrossChainUTXO {
 export type WalletType = 'backing' | 'cold' | 'custodial' | 'custodialPaired' | 'hot' | 'trading';
 export type SubWalletType = 'distributedCustody';
 
+export type WalletUser = {
```

Comment:
Can we pull this from `@bitgo/public-types`. If not, we may want to add it to provide full types.

Reply:
+1